opm/parser/eclipse/EclipseState/Schedule/DynamicState.hpp

```diff
@@ -24,6 +24,7 @@
 #include <stdexcept>
 #include <vector>
 #include <algorithm>
+#include <functional>
```

Comment:
Just out of curiosity, what part of `DynamicState.hpp` depends on facilities provided by `<functional>`?

Reply:
`std::function` - which was part of the PR at stage; removed now